tests/util/iperf_proc.py

```diff
@@ -0,0 +1,196 @@
+"""
+Iperf network performance testing module.
+Used by tests for simple bandwidth measurements; uses iperf.
+
+"""
+
```

Comment:
Curious if you looked around for an existing Python wrapper for Iperf before writing this one?